JavaScript-Runtime-(JSRT)-Overview.md

```diff
@@ -85,9 +85,116 @@ All other supported ES2015 features do not require any setup to run in JSRT.
 
 Features based on promises, such as [ES2016/ES7 Async Functions](https://blogs.windows.com/msedgedev/2015/09/30/asynchronous-code-gets-easier-with-es2016-async-function-support-in-chakra-and-microsoft-edge/), would require setting up promises before usage. 
 
+## Modules
+ES6 Modules are supported in ChakraCore. However, the host needs to specify how to resolve a module specifier since this part is host-specific not runtime-related. JSRT provides experimental APIs to set up ES6 modules. This [PR](https://github.com/Microsoft/ChakraCore/pull/1254) gives you an overview of the related APIs and an example implementation (see WScriptJsrt.cpp) in the ch test host. Better documentation will be provided in the future.
+
 ## Experimental Features
 Experimental JavaScript features that are not yet stable are hidden behind an experimental flag. You can enable these features on a runtime by specifying **JsRuntimeAttributeEnableExperimentalFeatures** flag when **JsCreateRuntime** is called. 
 
+## Basic tasks
+
+This section aims to illustrate how to achieve a few common tasks with JSRT with code samples.
+
+### Expose native functions/objects to JavaScript
+
+The basic idea is to create a native function or (external) object, and expose it to JS by setting it as a property of an object JS is already aware of. For example, let's expose `console.log`, which many JavaScript hosts expose but isn't part of JavaScript.
+
+```cpp
+// a native function for logging
+JsValueRef CALLBACK LogCB(JsValueRef callee, bool isConstructCall, JsValueRef *arguments, unsigned short argumentCount, void *callbackState)
+{
+    for (unsigned int index = 1; index < argumentCount; index++)
+    {
+        if (index > 1)
+        {
+            wprintf(L" ");
+        }
+        JsValueRef stringValue;
+        JsConvertValueToString(arguments[index], &stringValue);
+        const wchar_t *string;
+        size_t length;
+        JsStringToPointer(stringValue, &string, &length);
```

Comment:
Do we want these examples to also work cross-plat?  For instance `JsStringToPointer` isn't exposed on Linux/Mac, instead JsCopyString needs to be used.